src/ipc/zone/client/client_trigger.rs

```diff
@@ -78,7 +79,20 @@ pub enum ClientTriggerCommand {
 
     /// The player begins an emote.
     #[brw(magic = 0x01F4u32)]
-    Emote { emote: u32 },
+    Emote {
+        /// The id of the emote.
+        emote: u32,
+        /// 0/false = text shown, 1/true = text hidden
+        #[brw(pad_before = 4)]
+        #[br(map = read_bool_from::<u32>)]
+        #[bw(map = write_bool_as::<u32>)]
+        hide_text: bool,
+        /// The actor id of the target.
+        #[brw(pad_before = 8)]
```

Comment:
If these paddings are empty, make sure to declare them so - just so we don't start wondering later!